db/census_participant.go

```diff
@@ -480,6 +480,47 @@ func (ms *MongoStorage) SetBulkCensusParticipant(
 	return progressChan, nil
 }
 
+func (ms *MongoStorage) setBulkCensusParticipant(
+	ctx context.Context, censusID string, memberIDs []primitive.ObjectID,
+) error {
+	currentTime := time.Now()
+	docs := make([]mongo.WriteModel, 0, len(memberIDs))
+	for _, pid := range memberIDs {
+		// Create participant filter and document
+		id := pid.Hex()
+		censusParticipantsFilter := bson.M{
+			"participantID": id,
+			"censusId":      censusID,
```

Comment:
ID vs Id

Reply:
yeah, this comes from the coresponding types
https://github.com/vocdoni/saas-backend/blob/78d2be9fdc2b46a8ef2e4b4b2e1545bc494be3d8/db/types.go#L266

The PR #164 handle the apicommon/types.go json cases but i see there are various inconsistencies on the bson part as well